Code/Tools/ProjectManager/Source/NewProjectSettingsScreen.h

```diff
@@ -25,7 +25,7 @@ namespace O3DE::ProjectManager
     class NewProjectSettingsScreen
         : public ProjectSettingsScreen
     {
-        Q_OBJECT
+        Q_OBJECT 
```

Comment:
revert this unnecessary change
```suggestion
        Q_OBJECT
```

Reply:
There is a signal (maybe added after this pull request) that needs this.